apps/charterafrica/src/components/FlourishChart/FlourishChart.js

```diff
@@ -9,9 +9,14 @@ const FlourishChart = React.forwardRef(function FlourishChart(props, ref) {
 
   useEffect(() => {
     const handleMessageEvent = (event) => {
-      const { sender, context, ...message } = JSON.parse(event.data) || {};
-      if (sender === "Flourish" && context === "iframe.resize") {
-        setHeight(Math.ceil(message.height));
+      try {
+        const { sender, context, ...message } = JSON.parse(event.data) || {};
+        if (sender === "Flourish" && context === "iframe.resize") {
+          return setHeight(Math.ceil(message.height));
+        }
+        return null;
+      } catch (error) {
```

Comment:
When will error be thrown?

Reply:
This was meant to fix an error if `event.data` was not valid JSON. It looks though that that is caused by browser extensions.

Reply:
Which browser extension and why do should we handle that in our codebase @kelvinkipruto ?

Reply:
I have reverted the change.